server/form-pages/apply/risk-and-need-factors/oasys-import/roshSummary.ts

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable no-underscore-dangle */
+import type { DataServices, PersonRisksUI } from '@approved-premises/ui'
+
+import type {
+  Application,
+  ArrayOfOASysRiskOfSeriousHarmSummaryQuestions,
+  OASysQuestion,
+  OASysSections,
+} from '@approved-premises/api'
+
+import TasklistPage from '../../../tasklistPage'
+
+import { Page } from '../../../utils/decorators'
+
+type RoshSummaryBody = {
+  roshAnswers: Array<string> | Record<string, string>
+  roshSummaries: ArrayOfOASysRiskOfSeriousHarmSummaryQuestions
+}
+
+@Page({
+  name: 'rosh-summary',
+  bodyProperties: ['roshAnswers', 'roshSummaries'],
+})
+export default class RoshSummary implements TasklistPage {
+  title = 'Edit risk information'
+
+  roshSummary: ArrayOfOASysRiskOfSeriousHarmSummaryQuestions
+
+  risks: PersonRisksUI
+
+  roshAnswers: Record<string, string>
+
+  constructor(public body: Partial<RoshSummaryBody>) {}
+
+  static async initialize(
+    body: Record<string, unknown>,
+    application: Application,
+    token: string,
+    dataServices: DataServices,
+  ) {
+    const oasysSections: OASysSections = await dataServices.personService.getOasysSections(
+      token,
+      application.person.crn,
+    )
+    const risks = await dataServices.personService.getPersonRisks(token, application.person.crn)
+
+    const roshSummary = oasysSections.roshSummary.sort((a, b) => Number(a.questionNumber) - Number(b.questionNumber))
+    const roshAnswers = body?.roshAnswers ? body.roshAnswers : []
+
+    body.roshSummaries = ((roshSummary || []) as Array<OASysQuestion>).map((summary, i) => {
+      return {
+        ...summary,
+        answer: roshAnswers[i],
+      }
+    })
+
+    const page = new RoshSummary(body)
+    page.roshSummary = roshSummary
+    page.risks = risks
+
+    return page
+  }
+
+  previous() {
+    return 'optional-oasys-sections'
+  }
+
+  next() {
+    return ''
+  }
+
+  roshTextAreas() {
+    return this.roshSummary
+      .map(roshQuestion => {
+        return `<div class="govuk-form-group">
+                <h3 class="govuk-label-wrapper">
+                    <label class="govuk-label govuk-label--m" for=roshAnswers[${roshQuestion.questionNumber}]>
+                        ${roshQuestion.label}
+                    </label>
+                </h3>
+                <textarea class="govuk-textarea" id=roshAnswers[${roshQuestion.questionNumber}] name=roshAnswers[${
+          roshQuestion.questionNumber
+        }] rows="8">${roshQuestion?.answer || ''}</textarea>
```

Comment:
I'm not sure what you mean by escaped here.
This will be the answer taken from OASys that can be edited by the user

Reply:
I mean the answer could be something like, "<b>Some answer", and you'd want that transformed to "&lt;b&gt;Some answer", so the user actually sees "<b>Some answer", rather than the rest of the page in bold

Reply:
Yeah, this is a good point. I think we have similar in the documents work as well, which we (I!) should fix